src/config.h

```diff
@@ -83,6 +83,8 @@ struct Config{
   Cron bgsave_cron;
   CompactionCheckerRange compaction_checker_range{-1, -1};
   std::map<std::string, std::string> tokens;
+  std::string rename_command_;
+  std::string rename_command_new_name_;
```

Comment:
Should we allow renaming more than one command?

Reply:
redis only allow rename one command ,  kvrocks allow rename multi command ?

Reply:
Redis allows renaming multi commands indeed